website/blog/2024-03-03-AutoGen-Update/index.mdx

```diff
@@ -0,0 +1,188 @@
+featured in a published case study, allowing students to practice their interview
+skills with realistic information.
+- Benjamin D Stern, MS, DPT, Assistant Professor, Doctor of Physical Therapy Program,
+Tufts University School of Medicine
+```
+```
+Autogen has been a game changer for how we analyze companies and products! Through
+collaborative discourse between AI Agents we are able to shave days off our research
+and analysis process.
+- Justin Trugman, Cofounder & Head of Technology at BetterFutureLabs
+```
+
+These are just a small fraction of examples. We have seen big enterprise customers’ interest from pretty much every vertical industry: Accounting, Airlines, Biotech, Consulting, Consumer Packaged Goods, Electronics, Entertainment, Finance, Fintech, Government, Healthcare, Manufacturer, Metals, Pharmacy, Research, Retailer, Social Media, Software, Supply Chain, Technology, Telecom…
+
+AutoGen is used or contributed by companies, organizations, universities from A to Z, in all over the world. We have seen hundreds of example applications. Some organization uses AutoGen as the backbone to build their agent platform. Others use AutoGen for diverse scenarios, including research and investment to novel and creative applications of multiple agents.
+
+## Milestones
+
+AutoGen has a large and active community of developers, researchers and AI practitioners.
+- 22K+ stars on [GitHub](https://aka.ms/autogen-gh), 3K+ forks
+- 14K+ members on [Discord](https://aka.ms/autogen-dc)
+- 100K+ downloads per months
+- 3M+ views on Youtube (400+ community-generated videos)
+- 100+ citations on [Google Scholar](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=IiSNwnAAAAAJ&citation_for_view=IiSNwnAAAAAJ:zCpYd49hD24C)
+
+I am so amazed by their creativity and passion.
+I also appreciate the recognition and awards AutoGen has received, such as:
+- Selected by [TheSequence: My Five Favorite AI Papers of 2023](https://thesequence.substack.com/p/my-five-favorite-ai-papers-of-2023)
+- Top trending repo on GitHub in Oct'23
+- Selected into [Open100: Top 100 Open Source achievements](https://www.benchcouncil.org/evaluation/opencs/annual.html) only 35 days after spinoff
+
+On March 1, the initial AutoGen multi-agent experiment on the challenging [GAIA](https://huggingface.co/spaces/gaia-benchmark/leaderboard) benchmark turned out to achieve the No. 1 accuracy with a big leap, in all the three levels.
+
+![gaia](img/gaia.png)
+
+That shows the big potential of using AutoGen in solving complex tasks.
+And it's just the beginning of the community's effort to answering a few hard open questions.
+
+## Open Questions
+
+In the [AutoGen technical report](https://arxiv.org/abs/2308.08155), we laid out a number of challenging research questions:
+
+1. How to design optimal multi-agent workflows?
+1. How to create highly capable agents?
+1. How to enable scale, safety and human agency?
```

Comment:
Hmm, strange that all three points start with 1.?

Reply:
I believe markdown automatically enumerates. So the final displayed index should be 1. 2. 3.